packages/run-protocol/src/runStake/params.js

```diff
@@ -0,0 +1,89 @@
+// @ts-check
+
+import {
+  CONTRACT_ELECTORATE,
+  makeParamManager,
+  ParamTypes,
+} from '@agoric/governance';
+
+/** RUNstakeParams are the parameters subject to governance. */
+export const RUNstakeParams = /** @type { const } */ ({
```

Comment:
I don't know if we have style guidelines on this already but I advocate that acronyms by treated as words when smooshed: `RunStakeParams`.

(though is RUN an acronym or just a haughty word?)

Reply:
not an acronym.

Reply:
Since we have `makeRunStakeParamManager` and friends, I'm demoting this to a local name (`PKey`).

The only import was a test, and the test should independently spell the keyword; the test should fail if we un-intentionally change the API.